cardano-cli/test/cardano-cli-golden/files/golden/help/key.cli

```diff
@@ -1,6 +1,8 @@
 Usage: cardano-cli key 
```

Comment:
I am wondering if we should have a new command `root-key` that has these commands underneath. We need to make a clear distinction between the root key and the derived keys. Putting the commands under `key` makes this distinction a little less clear IMO.

Reply:
So far we are not really letting the user handle root keys, we are deriving normal extended keys directly from the mnemonic. But if we plan on adding support for deriving root keys too, and/or other commands specific for root keys then it would make sense.